[2024-12-17T11:14:17.689+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-17T11:14:17.697+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: call_9851100067_2024-12-17T11-12-05-954056Z.branch_on_call_status scheduled__2024-12-17T11:12:05.954056+00:00 [queued]>
[2024-12-17T11:14:17.700+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: call_9851100067_2024-12-17T11-12-05-954056Z.branch_on_call_status scheduled__2024-12-17T11:12:05.954056+00:00 [queued]>
[2024-12-17T11:14:17.700+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-17T11:14:17.705+0000] {taskinstance.py:2889} INFO - Executing <Task(_BranchPythonDecoratedOperator): branch_on_call_status> on 2024-12-17 11:12:05.954056+00:00
[2024-12-17T11:14:17.710+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=140452) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-17T11:14:17.711+0000] {standard_task_runner.py:72} INFO - Started process 140453 to run task
[2024-12-17T11:14:17.711+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'call_9851100067_2024-12-17T11-12-05-954056Z', 'branch_on_call_status', 'scheduled__2024-12-17T11:12:05.954056+00:00', '--job-id', '685', '--raw', '--subdir', 'DAGS_FOLDER/complete_dag.py', '--cfg-path', '/tmp/tmpohcnhur9']
[2024-12-17T11:14:17.712+0000] {standard_task_runner.py:105} INFO - Job 685: Subtask branch_on_call_status
[2024-12-17T11:14:17.736+0000] {task_command.py:467} INFO - Running <TaskInstance: call_9851100067_2024-12-17T11-12-05-954056Z.branch_on_call_status scheduled__2024-12-17T11:12:05.954056+00:00 [running]> on host 1868de310944
[2024-12-17T11:14:17.784+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='call_9851100067_2024-12-17T11-12-05-954056Z' AIRFLOW_CTX_TASK_ID='branch_on_call_status' AIRFLOW_CTX_EXECUTION_DATE='2024-12-17T11:12:05.954056+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-17T11:12:05.954056+00:00'
[2024-12-17T11:14:17.784+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-17T11:14:17.785+0000] {python.py:240} INFO - Done. Returned value was: find_reason_for_unanswered
[2024-12-17T11:14:17.785+0000] {branch.py:38} INFO - Branch into find_reason_for_unanswered
[2024-12-17T11:14:17.786+0000] {skipmixin.py:230} INFO - Following branch ('find_reason_for_unanswered',)
[2024-12-17T11:14:17.791+0000] {skipmixin.py:278} INFO - Skipping tasks [('find_if_proper_interaction_is_completed', -1)]
[2024-12-17T11:14:17.806+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-17T11:14:17.806+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=call_9851100067_2024-12-17T11-12-05-954056Z, task_id=branch_on_call_status, run_id=scheduled__2024-12-17T11:12:05.954056+00:00, execution_date=20241217T111205, start_date=20241217T111417, end_date=20241217T111417
[2024-12-17T11:14:17.844+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-17T11:14:17.859+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-17T11:14:17.860+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
